hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/PerformanceEvaluation.java

```diff
@@ -1998,6 +2028,45 @@ protected void testTakedown() throws IOException {
       super.testTakedown();
     }
   }
+  static class MetaRandomReadTest extends MetaTest {
+    private Random rd = new Random();
+    private RegionLocator regionLocator;
+
+    MetaRandomReadTest(Connection con, TestOptions options, Status status) {
+      super(con, options, status);
+      LOG.info("call getRegionLocation");
+    }
+
+    @Override
+    void onStartup() throws IOException {
+      super.onStartup();
+      this.regionLocator = connection.getRegionLocator(table.getName());
+    }
+
+    @Override
+    boolean testRow(final int i, final long startTime) throws IOException, InterruptedException {
+      if (opts.randomSleep > 0) {
+        Thread.sleep(rd.nextInt(opts.randomSleep));
+      }
+      HRegionLocation hRegionLocation = regionLocator.getRegionLocation(
+        getSplitKey(rd.nextInt(opts.perClientRunRows)), true);
+      if (LOG.isDebugEnabled()) {
```

Comment:
No need of the if debug test... use parameterized logging... "Get location for {}", hRegionLocation.... no need of 'region' in log message.